website/content/en/scripts/step10-deploy.sh

```diff
@@ -0,0 +1,6 @@
+kubectl create namespace karpenter
```

Comment:
Did you mean to include this? This must already exist when we do the helm install.

Reply:
Seems like the previous step creates a helm template then applies it with kubectl. That sounds like we need this line.

Reply:
Should I remove the line or not?

Reply:
I removed this line and replaced and added --create-namespace to step01. Let me know if I need to do more than that.